modules/hcp-ocp-differences.adoc

```diff
@@ -0,0 +1,168 @@
+
+|===
+
+[id="operators-and-control-plane_{context}"]
+== Operators and control plane
+
+[cols="2a,4a",options="header"]
+|===
+
+|{product-title} |{hcp}
+
+|A standalone {product-title} cluster contains separate Operators for each control plane component.
+|A hosted cluster contains a single Operator named Control Plane Operator that runs in the hosted control plane namespace on the management cluster.
+
+|etcd uses storage that is mounted on the control plane nodes. The etcd cluster Operator manages etcd.
+|etcd uses a (PVC) for storage and is managed by the Control Plane Operator.
+
+|The Ingress Operator, network related Operators, and {olm-first} run on the cluster.
+|The Ingress Operator, network related Operators, and {olm-first} run in the hosted control plane namespace on the management cluster.
+
+|The OAuth server runs inside the cluster and is exposed through a route in the cluster.
+|The OAuth server runs inside the control plane and is exposed either through a route, node port, or load balancer on the management cluster.
+
+|===
+
+[id="upgrades_{context}"]
+== Updates
+
+[cols="2a,2a",options="header"]
+|===
+
+|{product-title} |{hcp}
+
+|The Cluster Version Operator (CVO) orchestrates the update process and monitors the `ClusterVersion` resource. Administrators and OpenShift components can interact with the CVO through the `ClusterVersion` resource. The `oc adm upgrade` command results in a change to the `ClusterVersion.Spec.DesiredUpdate` field in the `ClusterVersion` resource.
+|The {hcp} update results in a change to the `.spec.release.image` field in the `HostedCluster` and `NodePools` resources. Any changes to the `ClusterVersion` resource are ignored.
+
+|After you update an {product-title} cluster, both the control plane and worker machines are updated.
+|After you update the hosted cluster, only the control plane is updated. You perform the node pool updates separately.
+
+|===
+
+[id="machine-config-manage_{context}"]
+== Machine configuration and management
+
+[cols="2a,2a",options="header"]
+|===
+
+|{product-title} |{hcp}
+
+|The `MachineSets` resource manages machines in the `openshift-machine-api` namespace.
+|The `NodePool` resource manages machines on the management cluster.
+
+|A set of control plane machines are available.
+|A set of control plane machines do not exist.
+
+|You enable a machine health check by using the `MachineHealthCheck` resource.
+|You enable a machine health check through the `.spec.management.autoRepair` field in the `NodePool` resource.
+
+|You enable autoscaling by using the `ClusterAutoscaler` and `MachineAutoscaler` resources.
+|You enable autoscaling through the `spec.autoScaling` field in the `NodePool` resource.
+
+|Machines and machine sets are exposed in the cluster.
+|Machines, machine sets, and machine deployments from upstream CAPI are used to manage machines but are not exposed to the user.
```

Comment:
```suggestion
|Machines, machine sets, and machine deployments from upstream CAPI are used to manage machines but are not exposed to the user.
```

I vaguely remember that we should use Cluster API as against CAPI but please check with Ben Scott.

Reply:
Ah, yes, an attribute exists. Use the attribute in place of CAPI:

![Screenshot from 2025-01-08 11-57-38](https://github.com/user-attachments/assets/16f4ab97-fcc1-4354-8f43-725588abdf9b)